zs/metadata.py

```diff
@@ -78,7 +78,7 @@
 }
 
 # Unaltered Eia861 archive.
-eia861_raw_uuid = "70999ef2-50e9-47ae-a4f6-5d69e6ff98d1"
+eia861_raw_uuid = "d6d05d7e-a0db-4346-a828-977d214f5aca"
```

Comment:
Actually we don't want to change the UUID -- right now it's the ID that links together different vintages of the archive on Zenodo. With a different UUID we would create a whole new separate collection of data.